guide/getting-started/software.md

```diff
@@ -65,11 +67,12 @@ Primarily used by developers, command-line interfaces typically offer the most f
    layout = "float-right-desktop"
 %}
 
-If a blockchain can be thought of as a public database, then explorers are simply windows into that database. They typically display the data structure and maybe offer some basic functionality like looking up specific transactions to verify that they were processed.
+If blockchain is a public database, block explorers are simply windows into that database. They let users view data about specific transactions, latest blocks, block height, and so on.
+
 
 </div>
 
-Explorers often also provide insight into activity on the bitcoin network, like the number of daily transactions. While not essential, it can be helpful to understand this activity as it may affect design decisions. For example, there are typically fewer transactions on weekends. This results in lower fees, which is ideal for low-priority transactions.
+Explorers also provide insight into bigger picture activity on the Bitcoin network, such as daily transaction numbers. For example, there are typically fewer transactions on weekends, which results in lower fees, which are ideal for low-priority transactions.
 
 ## Wallets
```

Comment:
How could we integrate Lightning into this section? Food for thought for another PR.